lib/peeps.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
+
+class Peeps
+
+  def self.all
+    if ENV['RACK_ENV'] == 'test'
```

Comment:
Might be good to extract this into a Database connection class which handles the connection to the database and the queries